File: Omkar's Pet Cat/chat1.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int T;
    cin >> T;

    while (T--) {
        int n;
        cin >> n;

        vector<long long> a(n);
        long long mn = LLONG_MAX;

        for (auto &x : a) {
            cin >> x;
            mn = min(mn, x);
        }

        int countMin = 0;
        bool hasNonMultiple = false;

        for (long long x : a) {
            if (x == mn) {
                countMin++;
            }

            if (x % mn != 0) {
                hasNonMultiple = true;
            }
        }

        if (countMin == 1 || hasNonMultiple) {
            cout << "YES\n";
        } else {
            cout << "NO\n";
        }
    }

    return 0;
}

File: Omkar's Pet Cat/chat1.py
import sys

def solve():
    input = sys.stdin.buffer.readline

    t = int(input())
    answers = []

    for _ in range(t):
        n = int(input())
        a = list(map(int, input().split()))

        mn = min(a)

        count_min = a.count(mn)
        has_non_multiple = any(x % mn != 0 for x in a)

        if count_min == 1 or has_non_multiple:
            answers.append("YES")
        else:
            answers.append("NO")

    sys.stdout.write("\n".join(answers))


if __name__ == "__main__":
    solve()